### PR TITLE
bug 1430661. Update masterConfig metricsPublicURL on install

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

**Comment by jcantrill** [comment]
@sdodson @ewolinetz Should we cherry-pick to 1.5?

**Comment by sdodson** [comment]
Yes please.

File: roles/openshift_metrics/tasks/update_master_config.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+- name: Adding metrics route information to metricsPublicURL
+  modify_yaml:
+    dest: "{{ openshift.common.config_base }}/master/master-config.yaml"
+    yaml_key: assetConfig.metricsPublicURL
+    yaml_value: "https://{{ openshift_metrics_hawkular_hostname}}"
```
Comment (ewolinetz) [inline roles/openshift_metrics/tasks/update_master_config.yaml]: just to make sure, that variable doesn't already include "https://" correct?

**Comment by ewolinetz** [comment]
This looks like it currently only updates for the master we are installing on, we want to make sure we also iterate over all the other masters as well and update their configs

File: roles/openshift_metrics/tasks/update_master_config.yaml (hunk 1 above)
Comment (jcantrill) [inline roles/openshift_metrics/tasks/update_master_config.yaml]: do we do that in logging? I did not see that.

**Comment by jcantrill** [comment]
@ewolinetz is there example where we iterate masters to update configs?

**Comment by ewolinetz** [comment]
@jcantrill https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/openshift_hosted.yml#L49-L62

**Comment by jcantrill** [comment]
Seems like there should be some optimizations here as in if I'm installing both logging in metrics, there should be no need to bump the master server twice.

**Comment by ewolinetz** [comment]
@jcantrill thats the beauty of a handler, it should only run it once as long as its installing it from the same playbook

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by stevekuznetsov** [comment]
@openshift-bot, the [last build](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_openshift_ansible/22/) failed from the following flakes:
 - [conformance](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_openshift_ansible/22/testReport/junit/(root)/Extended/_builds__Conformance__oc_new_app_should_succeed_with_a___name_of_58_characters/) failed on: https://github.com/openshift/origin/issues/13067

re[merge]

**Pull request merged**